prow/config/jobs/istio.yaml

```diff
@@ -0,0 +1,257 @@
+repo: istio/istio
+branches:
+  - master
```

Comment:
Are you going to add new branches here? 
What will happen when we add more tests to master, but not to another branch?

Reply:
I think we will probably need to fork the config, which isn't great. I couldn't think of any better ideas, do you have any?